src/main/java/com/documents/controllers/SecretaryController.java

```diff
@@ -0,0 +1,85 @@
+package com.documents.controllers;
+
+import com.documents.models.Secretary;
+import com.documents.services.SecretaryService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Created by Georgiana&Ecaterina on 07.05.2017.
+ */
+@RestController
+@RequestMapping(value = "/secretary")
+
+public class SecretaryController {
+
+    @Autowired
+    private SecretaryService secretaryService;
+
+    @RequestMapping(value = "/secretary/{id}", method = RequestMethod.GET)
```

Comment:
Because you added the  `_@RequestMapping(value = "/secretary")_` at the beginning of the class, the `_/secretary_` is now the implicit path, therefore you don't need to write it again. 
 _@RequestMapping(value = "/{id}", method = RequestMethod.GET)_ -> and the path will be  `_/secretary/{id}_`